mlflow/ml-package-versions.yml

```diff
@@ -714,7 +714,10 @@ llama_index:
     maximum: "0.10.53"
     requirements:
       ">= 0.0.0": ["llama-index", "openai"]
-    run:
+    run: |
+      # Install the same version of llama-index-core as llama-index
```

Comment:
note: LlamaIndex doesn't pin the code package somehow, but version inconsistency between two packages causes errors indeed.